homeassistant/components/bmw_connected_drive.py

```diff
@@ -4,30 +4,28 @@
 For more details about this platform, please refer to the documentation at
 https://home-assistant.io/components/bmw_connected_drive/
 """
-import logging
 import datetime
-
+import logging
 import voluptuous as vol
```

Comment:
There should be a blank line here between logging and voluptuous.

Reply:
fixed